docs/CHANGES.md

```diff
@@ -0,0 +1,228 @@
+# v1.0 CHANGES
+
+In Godwoken v0, Godwoken EVM has using an internal address format, users need to integrate web3-provider to handling the conversion between Godwoken internal address and the Ethereum address. Even in most cases the conversion is handled transparently, this incompatiblity still causes some problem in conner cases. Such as, we cannot hanlde the conversion happend inside metamask or some hardware wallets environments because the web3-provider plugin in unsupported in these environments.
+
+In the v1 version, we add a new builtin contract: Ethereum address registry, when a new account is created by user deposit, an Ethereum address is inserted to the contract. With the new builtin contract, the Ethereum address is supported in the EVM directly, now users can submit their transaction to Godwoken without using the web3-provider plugin. We suppose to provide 100% compatibility from the RPC level, once users switch the network to Godwoken, their can using the Ethereum Dapps or toolchains without modification.
```

Comment:
```suggestion
In the v1 version, we add a new builtin contract: Ethereum address registry. When a new account is created by user deposit, an Ethereum address is inserted to the contract. With the new builtin contract, the Ethereum address is supported in the EVM directly. Now users can submit their transaction to Godwoken without using the web3-provider plugin. We aim to provide 100% compatibility at the RPC level. When users switch to the Godwoken network, they can use Ethereum Dapps or toolchains without modification.
```

Also, would it be too strong to claim “without modification”? Maybe “with minimal modifications”?